templates/data/mysql.html

```diff
@@ -1,477 +1,533 @@
 {% extends 'base_index.html' %}
 
-{% block meta_copydoc %} https://docs.google.com/document/d/1KQmCfb4ITlYHOAEoHUwxHExf73qkMSWsLHnFPwFWFag/edit {% endblock meta_copydoc %}
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1KQmCfb4ITlYHOAEoHUwxHExf73qkMSWsLHnFPwFWFag/edit
+{% endblock meta_copydoc %}
 
-{% block title %}Enterprise-grade solutions for MySQL databases{% endblock %}
+{% block title %}An advanced, enterprise-grade MySQL solution from Canonical{% endblock %}
 
-{% block meta_description %}Get secure and automated deployment, maintenance and upgrades for your MySQL databases across private and public clouds.{% endblock %}
+{% block meta_description %}
+  Enterprise-grade MySQL with up to 10 years of support and security maintenance. It helps you secure and automate the deployment, maintenance and upgrades of your MySQL databases across private and public clouds.
+{% endblock %}
 
-{% block body_class %}is-paper{% endblock body_class %}
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
 {% block content %}
 
-<section class="p-strip is-shallow u-no-padding--bottom">
-  <div class="p-section">
-    <div class="row--50-50">
-      <div class="col">
-        <div class="p-section--shallow">
-          <h1 class="u-no-margin--bottom">MySQL operations, simplified</h1>
-        </div>
-        <div class="p-section--shallow">
-          <p>Secure and automate the deployment, maintenance and upgrades of your MySQL databases across private and public clouds.</p>
-        </div>
-        <hr>
-        <a class="p-button--positive" aria-controls="data-relational-dbs-modal" href="/contact-us">Contact us</a>
-      </div>
-      <div class="col">
-        <div class="u-vertically-center u-aspect-ratio--16-9 p-image-wrapper u-container--background u-align--center p-strip">
+  <section class="p-section--hero">
```

Comment:
Please use the new hero layout from vanilla here: https://vanillaframework.io/docs/patterns/hero#2575-signpost It perfectly matches the pattern

Reply:
This would create an unwanted layout on medium screens

Reply:
I believe design will be better to confirm it here. @mattea-turic